test/feature/dynamic_types/DynamicTypesTests.cpp

```diff
@@ -13073,7 +13073,7 @@ TEST_F(DynamicTypesTests, DynamicType_XML_struct_with_boolean)
     ASSERT_EQ(RETCODE_OK,
             DomainParticipantFactory::get_instance()->load_XML_profiles_file(DynamicTypesTests::config_file()));
 
-    DynamicType::_ref_type xml_type;
+    DynamicTypeBuilder::_ref_type xml_type;
```

Comment:
Please, update the rest of the file
```suggestion
    DynamicTypeBuilder::_ref_type xml_type_builder;
```